languages/javascript/scripts/scaffold/template/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
Is this the exact same we already use? If yes, you can just thumbs up this and resolve.